src/include/sof/audio/generic_processor/hifi_codec/public.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporatiogit an. All rights reserved.
```

Comment:
2020

Reply:
Btw, is this the header to the external binary. If so we should state here that any types/values are immutable.

Reply:
Further, typo (a "git a" somehow made its way in the middle of the line)

Reply:
I would try to find a fancy name instead of public.h. Maybe api.h or codec_interface.h?

Reply:
Intel Corporatiogit an.

?